### ISSUE TITLE
Connectors self managed config is wrong/misleading

### ISSUE THREAD

**Issue by zalanszak** [camunda/camunda-docs#4618]
here is the connectors config doc page:
https://docs.camunda.io/docs/self-managed/connectors-deployment/connectors-configuration/#secure-connection

It states that the env var CAMUNDA_IDENTITY_CLIENT_SECRET=<YOUR_OPERATE_CLIENT_SECRET>
However CAMUNDA_IDENTITY_CLIENT_SECRET should be the connectors client secret. 

Please correct this.

**Comment by akeller** [comment]
@sbuettner can you quickly confirm so we can get this PR'd?

**Comment by sbuettner** [comment]
Yeah, its confusing as its client that will be used to talk to operate.

**Comment by akeller** [comment]
Is `CAMUNDA_IDENTITY_CLIENT_SECRET=<CONNECTORS_CLIENT_SECRET>` valid?
